### PR TITLE
Landing page redesign feedback

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Applied design changes as outlined on mock-up
- Drive by:
  - Removed unused class on /hpe
  - Ran djlint 
  - Upgraded to latest vanilla (v4.13)

## QA

- View the site locally in your web browser at: https://ubuntu-com-13955.demos.haus/
- Check against [design](https://www.figma.com/design/Le4TTzyOFbcbqTPJ1tVoZT/U.com---Home-and-footer-(rebranding)?node-id=2158-5048&m=dev) and [doc](https://docs.google.com/document/d/1ySJxQbqVdeH4Tra0zwBm2Tn0s56kFGnEF7d8xDRTxwU/edit)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-11309

### CODE REVIEW

**Comment by lyubomir-popov** [comment]
ca nwe make this image touch the bottom? 
image

**Comment by lyubomir-popov** [comment]
image
some misalignments (missing nudge?)

File: templates/hpe/index.html [hunk 1]
```diff
@@ -1,448 +1,558 @@
...
+      <div class="col-3 col-medium-2">
+        {{
+        image(
         url="https://assets.ubuntu.com/v1/0ad256cf-hpe_pri_grn_pos_rgb.png",
         alt="",
         width="157",
         height="66",
         hi_def=True,
         loading="auto",
-        attrs={"class": "u-aspect-ratio--2-3"}
```
Comment (mtruj013) [inline templates/hpe/index.html:22]: Change here

File: templates/hpe/index.html [hunk 2]
```diff
@@ -1,448 +1,558 @@
...
+        <div class="u-image-separation">
+          {{
+          image(
           url="https://assets.ubuntu.com/v1/941754b0-dell server image1.png",
           alt="",
           width="1614",
           height="1076",
           hi_def=True,
           loading="lazy",
-          attrs={"class": "u-aspect-ratio--2-3"}
```
Comment (mtruj013) [inline templates/hpe/index.html:136]: Change here

File: templates/hpe/index.html [hunk 3]
```diff
@@ -1,448 +1,558 @@
...
+            style="background-color: #fff">
+        {{
+        image(
         url="https://assets.ubuntu.com/v1/5b5a466c-dell server image-2.png",
         alt="",
         width="2438",
         height="840",
         hi_def=True,
         loading="lazy",
-        attrs={"class": "u-aspect-ratio--2-3"}
```
Comment (mtruj013) [inline templates/hpe/index.html:257]: Change here

File: templates/hpe/index.html [hunk 4]
```diff
@@ -1,448 +1,558 @@
...
+        <div class="u-image-separation">
+          {{
+          image(
           url="https://assets.ubuntu.com/v1/62b49438-Rectangle 558-1.png",
           alt="",
           width="1184",
           height="711",
           hi_def=True,
           loading="lazy",
-          attrs={"class": "u-aspect-ratio--2-3"}
```
Comment (mtruj013) [inline templates/hpe/index.html:301]: Change here

File: templates/hpe/index.html [hunk 5]
```diff
@@ -1,448 +1,558 @@
...
+        <div class="u-image-separation">
+          {{
+          image(
           url="https://assets.ubuntu.com/v1/bf7d0182-Rectangle 558-2.png",
           alt="",
           width="1184",
           height="709",
           hi_def=True,
           loading="lazy",
-          attrs={"class": "u-aspect-ratio--2-3"}
```
Comment (mtruj013) [inline templates/hpe/index.html:373]: Change here

File: templates/hpe/index.html [hunk 6]
```diff
@@ -1,448 +1,558 @@
 {% extends "hpe/_base_hpe.html" %}
```
Comment (mtruj013) [inline templates/hpe/index.html:1]: Large diff only due to linting, please see comments in below where I've flagged the minor changes

**Comment by lyubomir-popov** [comment]
- no need for the p-section class on this paragraph
image
- we need to replace this image with one that has 2.4v aspect ratio
image
- no need for thess p-section--shallow classes, adds too much space there:
image
(review not finished)

File: static/sass/_pattern-image-ratios.scss [hunk 7]
```diff
@@ -0,0 +1,33 @@
+// XXX: MPT: Can be removed once this is live on Vanilla
```
Comment (bartaz) [inline static/sass/_pattern-image-ratios.scss:1]: Not needed anymore with 4.13

**Comment by lyubomir-popov** [comment]
@mtruj013 can you pls cherrypick the last commit from [this branch](https://github.com/canonical/ubuntu.com/tree/landing-refresh-lyubo) - that's the visual QA.

File: templates/base_index.html [hunk 8]
```diff
@@ -1419,10 +1105,10 @@ <h2 class="p-heading--1 u-no-margin--bottom">Multi-cloud applications&nbsp;&ndas
       </ul>
```
Comment (britneywwc) [inline templates/base_index.html:1105]: Screenshot 2024-06-21 at 12 00 49 PM
There is some top padding on the second row of list items. Can you please add "u-no-padding--top" to the list items on lines 1097 and 1100?

File: templates/base_index.html [hunk 9]
```diff
@@ -359,72 +344,40 @@ <h2 class="p-heading--1 u-no-margin--bottom">Significant enterprise savings</h2>
...
-          <div class="row">
-            <div class="col-3 col-start-large-4">
-              <p class="u-no-margin--bottom">
-                <a href="/contact-us" class="js-invoke-modal">Let's talk open source&nbsp;&rsaquo;</a>
-              </p>
-            </div>
-          </div>
+          <hr class="is-muted" />
+          <p class="u-no-margin--bottom">
+            <a href="/contact-us" class="js-invoke-modal p-button is-dark">Let's talk open source&nbsp;&rsaquo;</a>
```
Comment (britneywwc) [inline templates/base_index.html:349]: The button text is showing link colours here, it should be showing normal text colours like other buttons
Screenshot 2024-06-21 at 12 09 33 PM

File: templates/base_index.html [hunk 10]
```diff
@@ -474,11 +427,8 @@ <h2 class="p-heading--1 u-no-margin--bottom">Public cloud optimisation</h2>
         <div class="col-3 col-start-large-7 col-medium-2 col-medium-start-3">
```
Comment (britneywwc) [inline templates/base_index.html:427]: There is some top padding on the last row of list items. Can you please add "u-no-padding--top" to the list items on lines 417 and 420?

**Review by britneywwc — CHANGES_REQUESTED** [review]
Thanks @mtruj013! I left a couple comments to address some missing/incorrect links and list stylings, besides that everything looks good!

File: templates/base_index.html [hunk 11]
```diff
@@ -488,85 +438,49 @@ <h2 class="p-heading--1 u-no-margin--bottom">Public cloud optimisation</h2>
...
 
     <section class="p-section">
-      <div class="row--50-50 p-section--shallow">
+      <div class="row--50-50-on-large p-section--shallow">
         <hr class="p-rule" />
         <div class="col">
-          <h2 class="p-heading--1 u-no-margin--bottom">Multi-cloud Kubernetes and containers</h2>
-        </div>
-        <div class="col">
+          <h2 class="p-heading--1 u-no-margin--bottom u-no-max-width">Multi-cloud Kubernetes and containers</h2>
```
[the remark this answers is not in the document]
Comment (mtruj013) [reply templates/base_index.html:448]: Nope, design is the source of truth for layout 🙂

**Review by britneywwc — APPROVED** [review]
LGTM!

**Pull request merged**